Docs/README.md

```diff
@@ -0,0 +1,3 @@
+### Production Deployment Considerations
+1. Multi-master etcd deployment
+1. Autogeneration of certificate keys
```

Comment:
Makes sense. The logic behind putting the note here was in thinking that we might capture some of our to-dos from our call; that these weren't notes ready to be bubbled up just yet.